ui/widgets/nuxeo-date.js

```diff
@@ -40,7 +40,9 @@ import './nuxeo-tooltip.js';
     static get template() {
       return html`
         <span id="datetime" hidden$="[[!datetime]]">[[formatDate(datetime, format)]]</span>
-        <nuxeo-tooltip for="datetime">[[formatDateTime(datetime, tooltipFormat)]]</nuxeo-tooltip>
+        <nuxeo-tooltip for="datetime" hidden$="[[_validateTootip(datetime,format,tooltipFormat)]]"
```

Comment:
nitpicking: I find the name `_validateTootip` misleading in the context of this binding. It gives the impression that we are hiding the tooltip when it "validates". Perhaps something like `_equalFormats`, or `_noTooltipNeeded`? :thinking:

Reply:
good suggestion, changed it to _sameFormat